src/codegen/debug.rs

```diff
@@ -351,9 +416,208 @@ impl<'ink> DebugBuilder<'ink> {
 
         Ok(())
     }
+
+    fn create_debug_location(
+        &self,
+        scope: DIScope<'ink>,
+        line: usize,
+        column: usize,
+    ) -> Result<DILocation, Diagnostic> {
```

Comment:
Result not required

Reply:
They should, I think I mostly forgot to remove / change some panics, I'll take another look

Reply:
Correction: I had not seen the methods this is on, this one was useless i removed the method